tests/vlan_test.py

```diff
@@ -134,6 +134,124 @@
+|           |                 | Ethernet16      | untagged       |             |
++-----------+-----------------+-----------------+----------------+-------------+
+|      1001 |                 | Ethernet20      | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+|      1002 |                 | Ethernet20      | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+|      2000 | 192.168.0.10/21 | Ethernet20      | tagged         | enabled     |
+|           | fc02:1011::1/64 | Ethernet24      | untagged       |             |
+|           |                 | Ethernet28      | untagged       |             |
++-----------+-----------------+-----------------+----------------+-------------+
+|      3000 |                 | Ethernet20      | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+|      4000 |                 | PortChannel1001 | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+"""
+
+test_config_add_del_add_vlans_and_add_vlans_member_except_vlan__after_del_member_output = """\
++-----------+-----------------+-----------------+----------------+-------------+
+|   VLAN ID | IP Address      | Ports           | Port Tagging   | Proxy ARP   |
++===========+=================+=================+================+=============+
+|      1000 | 192.168.0.1/21  | Ethernet4       | untagged       | disabled    |
+|           | fc02:1000::1/64 | Ethernet8       | untagged       |             |
+|           |                 | Ethernet12      | untagged       |             |
+|           |                 | Ethernet16      | untagged       |             |
++-----------+-----------------+-----------------+----------------+-------------+
+|      1001 |                 | Ethernet20      | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+|      1002 |                 | Ethernet20      | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+|      2000 | 192.168.0.10/21 | Ethernet24      | untagged       | enabled     |
+|           | fc02:1011::1/64 | Ethernet28      | untagged       |             |
++-----------+-----------------+-----------------+----------------+-------------+
+|      3000 |                 |                 |                | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+|      4000 |                 | PortChannel1001 | tagged         | disabled    |
++-----------+-----------------+-----------------+----------------+-------------+
+"""
+
+test_config_add_del_vlan_and_vlan_member_with_switchport_modes_output = """\
++-----------+-----------------+-----------------+----------------+-------------+
```

Comment:
We have a dhcp helper column in current vlan show output? How is this handled? Also, can you please move the tagging to last column?

Reply:
This Dhcp helper column is added in vlan brief output outside of the utilities container (buildimage). Since our proposed changes have not changed output of "show vlan brief" command, we are not modifying existing behavior of VLANs.  Scope of these test cases is only within the utilities container, hence the DHCP column is not added.

For the tagging column, the order of columns inside "show vlan brief" command is defined in show/vlan.py file (line 101). We have not modified the order of columns or the "show vlan brief" command, this behavior is pre-existing.